01-introduction.md

```diff
@@ -48,17 +48,18 @@ Much of the research in education has rightly focused on people without these pr
 
 ## Instructional Design
 
-While what we know about educational psychology can constrain teaching,
-it doesn't dictate it.
-In particular,
-there are often several plausible approaches to teaching
-that are consistent with what we know about how brains learn.
-In order to decide between them,
-we also need to explore *instructional design*,
-which is the study of how to create learning materials.
-If educational psychology is the science,
-ID is the engineering:
-it's how our scientific understanding is put into practice on the ground.
+While what we know about educational psychology can constrain teaching, we are
+often challenged to accomodate our knowledge from the study of psychology with
+the evidence from instructional practice, and with influence stemming from
+instructor ideology.
+
+This is because, in some cases, there might be a few different approaches to
+teaching that could be used. Making out which of these is consistent with what
+we know about how brains learn can be difficult, and might appear to clash with
+educational practitioners' ideology.
+
+If educational psychology is the science, ID is the engineering: it's how our
```

Comment:
On Wed, Jan 13, 2016 at 8:26 PM, W. Trevor King <notifications@github.com>
wrote:

> In 01-introduction.md
> <https://github.com/swcarpentry/instructor-training/pull/61#discussion_r49686623>
> :
>
> > -we also need to explore *instructional design*,
> > -which is the study of how to create learning materials.
> > -If educational psychology is the science,
> > -ID is the engineering:
> > -it's how our scientific understanding is put into practice on the ground.
> > +While what we know about educational psychology can constrain teaching, we are
> > +often challenged to accomodate our knowledge from the study of psychology with
> > +the evidence from instructional practice, and with influence stemming from
> > +instructor ideology.
> > +
> > +This is because, in some cases, there might be a few different approaches to
> > +teaching that could be used. Making out which of these is consistent with what
> > +we know about how brains learn can be difficult, and might appear to clash with
> > +educational practitioners' ideology.
> > +
> > +If educational psychology is the science, ID is the engineering: it's how our
>
> I see no other references to “ID”. Certainly not enough to be worth
> coining an acronym for typing/reading efficiency.
>
> I think that the acronym is useful, so that instructors can use it
verbally when going over these materials.



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/instructor-training/pull/61/files#r49686623>
> .
>